practitioner-service/README.md

```diff
@@ -0,0 +1,77 @@
+# Practitioner Service
+Practitioner service validates the input person's details and adds it as a practitioner entity to the FHIR store.
+
+## API Endpoints
+### Create Practitioner
+Creates a new practitioner entity.
+
+**URL** : `/practitioner`
```

Comment:
what about adding a uuid optional param for update?

Reply:
this would be another endpoint (PUT) which is not required at the moment. suggest that when it is required, we'll do it on another PR?